audio/sequence_table.c

```diff
@@ -0,0 +1,4583 @@
+#include "global.h"
+#include "sound.h"
+
+u32 sequence_table_size = 1923;
```

Comment:
judging by this file there seems to have been a conflict with generic's recent pr, i think you need to merge changes from there